src/main/scala/io/sdkman/changelogs/MicronautMigrations.scala

```diff
@@ -0,0 +1,20 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "009")
+class MicronautMigrations {
+
+  @ChangeSet(order = "001", id = "001_add_micronaut_1_0_0_M1", author = "jameskleeh")
+  def migration001(implicit db: MongoDatabase) = {
+    insertCandidate(
+      Candidate(
+        candidate = "micronaut",
+        name = "Micronaut",
+        description = "Micronaut is an open source microservice framework for the JVM",
```

Comment:
Can we be a bit more elaborate with the description? This is what will appear on the [sdks page](http://beta.sdkman.io/sdks) of our site, as well ash when you do an `sdk ls` on the command line.

Reply:
@marc0der Thank you for letting me know. I wasn't aware of that. I'll update it